scripts/check_plugins_images.sh

```diff
@@ -1,34 +0,0 @@
-#!/bin/bash
```

Comment:
why delete this check script?

Reply:
Because I included the check it does in the schema, so it's covered by the regular validation. Sorry that's not clear at all, should have included a comment.

Reply:
For reference, it should be [this line](https://github.com/eclipse/che-plugin-registry/pull/205/files#diff-672f4dbb06abfc6f01b3ac2e267a3e4cR29)